### DISCUSSION TITLE
Can SwiftDialog accept a file?

### DISCUSSION THREAD

**Discussion by BleakleyMcD** [swiftDialog/swiftDialog#357] [Q&A] [answered]
Is it possible to point to a file in SwiftDialog? Or, put another way, it is possible for SwiftDialog to access the Finder? Could I, for instance, use SwitftDialog to point to a directory in which to save log files?

**Comment by bartreardon** [comment]
I'm not sure what you mean. There's a file option on a textfields https://github.com/swiftDialog/swiftDialog/wiki/Textfields#fileselect-and-filetype

**Comment by BleakleyMcD** [comment]
Ah, ok. I guess I missed that! follow up - is there the ability to choose/set a directory to which files generated by swiftdialog are saved?

Thank you! I love this tool!

**Comment by bartreardon** [comment]
for logs? no - other than that swiftdialog doesn't generate any files. All output is sent to stdout and you can do with that data what you will.

**Comment by BleakleyMcD** [comment]
yeah for logs for similar. Ok. I know GTK or Pashua can be configured where the user chooses a destination via a finder-like gui where logs or other files created by processes triggered by choices made in the dialog will be saved.

could be a worthwhile feature to add to SwitftDialog?

**Comment by bartreardon** [comment, marked as answer]
yeah, that's not what swiftDialog does. any output (e.g. user selections or other input) is sent to stdout and can be collected and processed by the script that called swiftDialog in the first place.
